### ISSUE TITLE
Docker env file does not accept all args

### ISSUE THREAD

**Issue by goose-ws** [the-paperless-project/paperless#535]
This may be intended rather than a bug, but hopefully not, or hopefully something I can be pointed at a line to change to edit this behavior.

My docker-compose.env looks like:

```
USERMAP_UID=1001
USERMAP_GID=1001
PAPERLESS_DISABLE_LOGIN="true"
PAPERLESS_INLINE_DOC="true"
PAPERLESS_FORGIVING_OCR="true"
PAPERLESS_DATE_ORDER="MDY"
```

However, at least `PAPERLESS_DISABLE_LOGIN` and `PAPERLESS_INLINE_DOC` are not behaving like they would on a bare metal install. I'm unsure if the latter two are or not, yet. Is there a way to get these options to work in a Docker build as they do in a bare metal build?

**Comment by goose-ws** [comment]
Today I learned. I'll give that a shot. Thank you!

**Comment by goose-ws** [comment]
Confirmed fixed, and I believe this also fixes a part of #430 

Thank you again!
